tests/system/test_session_api.py

```diff
@@ -2521,6 +2521,44 @@ def test_partition_query(sessions_database, not_emulator):
     batch_txn.close()
 
 
+def test_mutation_groups_insert_or_update_then_query(sessions_database):
```

Comment:
Emulator tests are failing as this feature is not implemented there.
Can you add `not_emulator` to skip emulatior.

Reply:
Done.